doc/api/vm.md

```diff
@@ -1573,6 +1573,78 @@ inside a `vm.Context`, functions passed to them will be added to global queues,
 which are shared by all contexts. Therefore, callbacks passed to those functions
 are not controllable through the timeout either.
 
+### Class: `NodeRealm`
+
+> Stability: 1 - Experimental. Use `--experimental-noderealm` CLI flag to enable this feature.
+
+<!-- YAML
+added: REPLACEME
+-->
+
+* Extends: {EventEmitter}
+
+A `NodeRealm` is effectively a Node.js environment that runs within the
+same thread.
+
+```mjs
+import { NodeRealm } from 'node:vm';
+const noderealm = new NodeRealm();
+const myAsyncFunction = noderealm.createImport(import.meta.url)('my-module');
```

Comment:
Could this be simplified to `noderealm.import('my-module')`? As in, are we able to infer the module parent (`import.meta.url`) from within `createImport`, and if so, is there ever a reason we wouldn’t want to import relative to that?

Reply:
I don't think retrieving a function caller's location reliably is possible.